config/application.rb

```diff
@@ -15,8 +15,5 @@ class Application < Rails::Application
     #
     # These settings can be overridden in specific environments using the files
     # in config/environments, which are processed later.
-    #
```

Comment:
any specific reason for removing these as part of this PR?

Reply:
i re-added them